SkeletalRepresentationInitializer/Logic/itkThinPlateSplineExtended.cxx

```diff
@@ -1,3 +1,19 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Portions (c) Copyright Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
```

Comment:
It may be sensible to also add a section like this one:

```
This file was originally developed by John Doe, AwesomeLab, Awesome's University
and was supported through XXXX.
```

See Slicer code base for examples